docs/UserGuide.md

```diff
@@ -86,7 +87,7 @@ A handy reference for more experienced users who just need to know the format of
 **:information_source: Notes about the command format:**<br>
 
 * Words in `UPPER_CASE` are the attributes to be filled in by you.<br>
-  e.g. in `personadd n/NAME`, the `NAME` attribute could be `personadd n/John Doe`.
+  e.g. in `add n/NAME`, the `NAME` attribute could be `add n/John Doe`.
```

Comment:
same as the comment below, this change is wrong. Did you rebase from master yet?